content/integrations/azure.md

```diff
@@ -18,25 +18,73 @@ sidebar:
+3. Click on "Applications" <img src="/static/images/azure/application_click.png" style="width: 50%;display:block;"/>
+4. Click "Add" <img src="/static/images/azure/add_application_icon.png" style="width: 50%;display:block;"/>
+5. Click "Add an application my organization is developing"
+6. Give it a name, i.e. Datadog Auth App
+7. Select “Web application and/or web api”
+8. Provide a sign-on URL and App ID URI
+	- These can be any value, they are not needed for authentication 
+	- i.e. http://mydatadogapp123.com	
+9. Click the check mark. You should be on a page that says "Your app has been added!"
+10. Click on "Configure" (Make sure this is configure for the app, not the active directory) <img src="/static/images/azure/configure_click.png" style="width: 50%;display:block;"/>
+11. Scroll down to Client ID. Copy and paste the value in the form on the Datadog setup page <img src="/static/images/azure/client_id_form.png" style="width: 50%;display:block;display:block;"/>
+12. Scroll down to Keys and create a new key (1 or 2 years are both acceptable)
+	- Note: the client secret key will be shown after you save
+13. Scroll down to "Permissions To Other Applications"
+14. Click Add Application
+15. Select Windows Azure Service Management API and press the check mark on the bottom right of the screen
+16. Under Delegated Permissions, check off “Access Azure Service Management as organization users (preview)” <img src="/static/images/azure/app_permissions_dropdown.png" style="width: 50%;display:block;"/>
+17. Click "Save" at the bottom of the screen 
+18. Scroll back up to Keys, and copy and paste the generated Client Secret key in the form on the Datadog setup page <img src="/static/images/azure/client_secret_form.png" style="width: 50%;display:block;display:block;"/>
+
+#### Giving Read Permissions to the Application
+
+1. Navigate to [portal.azure.com][2]
+2. Navigate to “Subscriptions” on the left hand menu <img src="/static/images/azure/subscriptions_icon.png" style="width: 25%;display:block;"/>
+3. Click on the subscription you would like to monitor
+4. Click on the user icon <img src="/static/images/azure/user_button.png" style="width: 50%;display:block;"/>
+5. Click "Add" User <img src="/static/images/azure/add_user_button.png" style="width: 25%;display:block;"/>
+6. Select “Reader” as a role <img src="/static/images/azure/reader_icon.png" style="width: 25%;display:block;"/>
+7. Search/select for the name of the Application you just created (i.e. Datadog Auth App)
+8. Click Select
+9. Press OK
+10. Repeat this process for any other subscriptions you would like to monitor
+
+**Navigate back to the Datadog setup page, and click "Update Configuration" to complete the installation process**
+
+**In a few minutes, metrics from applications under that subscription will begin to appear!**
+
 
-To setup Azure monitoring, go to [Azure integrations][1] and follow instructions on the page.
 
 ![Azure default dashboard](/static/images/azure.png)
```

Comment:
This should read  call out "Azure VM Default Dashboard"  or move into a dedicated VM doc